### ISSUE TITLE
make CEPC consequence section more robust

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/PWETF#193]
What happens when someone violates CEPC?

See https://geekfeminismdotorg.wordpress.com/about/code-of-conduct/ and https://www.contributor-covenant.org/version/2/1/code_of_conduct/ for reference.

See WIP https://github.com/w3c/PWETF/blob/main/CEPCdisciplinary-process.md 

Should these be separate documents?

**Comment by OR13** [comment]
Yes, this section feels very much missing from: https://www.w3.org/Consortium/cepc/

> Should these be separate documents?

No, I think the XOXO link you provided is a good format.

**Comment by jeffjaffe** [comment]
Providing current guidebook for banning from groups - for reference.

https://www.w3.org/Guide/process/banning.html

**Comment by mnot** [comment]
FWIW, I was very confused that consequences were seemingly missing from this document when I first read it -- it seemed toothless. Only when I found the very obscure reference to the [guidelines](https://www.w3.org/Guide/process/suspension.html) did it make sense.

It'd be good to get these all under one 'roof', or at the very least create a 'consquences' section to contain that link. Also, it'd be good to highlight the relationship with the process.

**Comment by TzviyaSiegman** [comment]
Thanks @mnot. I raised this at https://github.com/w3c/PWETF/issues/3. I agree with you. Historically we were advised to keep the documents separate so that we could update who the ombuds are without updating the Code. IDK if that is the advice we need to follow today. 

However, we just did a call for consensus on the code of conduct, and we are working on a major overhaul of the ombuds/mediator program. I recommend that we make this change in 2024 after we have the reporting structure in place for a bit. I'm happy to work with the board on the policy aspect of this.

**Comment by mnot** [comment]
> However, we just did a call for consensus on the code of conduct

Where is that?

**Comment by TzviyaSiegman** [comment]
@mnot I should clarify that this is a CfC within PWE prior to sending to AB/AC for final publication https://lists.w3.org/Archives/Public/public-pwe/2023Jun/0005.html. We gave a heads up at the AC meeting in May.

**Comment by mnot** [comment]
Thanks @TzviyaSiegman. The currently operative document says:

> This document has been reviewed by W3C Members and is endorsed by the Director as the W3C Code of Ethics and Professional Conduct (CEPC).

What will the new one say, under Director-free? Also, what does 'reviewed by W3C Members' mean specifically?

I ask because this document has the flavour of a policy, and is referenced as such in a few places (including the Process and the Member Agreement).

**Comment by TzviyaSiegman** [comment]
Thanks for pointing out the need to change the part about Director, @mnot. @wareid and I will work on a PR. 

"Reviewed by the members" means that the document was approved by the AC. This version will go through AC (and AB) review as well, much like the Process document. Perhaps it is unusual to require consensus on something that is roughly policy. I would love to discuss with you the extent of the authority of PWE in another space.

Previous versions of the CoC were reviewed by W3C Council from a legal perspective. This version was last reviewed by Wendy S (informally) in Dec 2022. @swickr suggested we might ask Council to review this iteration. @wareid and I would provide background.

**Comment by wareid** [comment]
@mnot We looked into this and that section was added as part of the publishing process, the editor's draft doesn't have that text, and it looks like other recently published documents don't have it either. We'll double check during publication though to be sure.
